include/SDL3/SDL_gpu.h

```diff
@@ -3715,6 +3715,23 @@ extern SDL_DECLSPEC bool SDLCALL SDL_GPUTextureSupportsSampleCount(
     SDL_GPUTextureFormat format,
     SDL_GPUSampleCount sample_count);
 
+/**
+ * Calculate the size in bytes of a texture format with dimensions.
+ *
+ * \param format a texture format.
+ * \param w width in pixels.
+ * \param h height in pixels.
+ * \param d depth in pixels.
+ * \returns the size of a texture with this format and dimensions.
+ *
+ * \since This function is available since SDL 3.2.0.
+ */
+extern SDL_DECLSPEC Uint32 SDLCALL SDL_CalculateGPUTextureFormatSize(
+    SDL_GPUTextureFormat format,
+    Uint32 w,
+    Uint32 h,
+    Uint32 d);
```

Comment:
```suggestion
    Uint32 depth);
```
It's 2024, we can afford extra characters for clarity. :)